installer/scripts/omsadmin.sh

```diff
@@ -787,6 +804,36 @@ show_workspace_status()
     fi
 }
 
+show_omsagent_proc_limit()
+{
+    llimit=-1
+    resultcode=0
+
+    count=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | wc -l`
+
+    if [ "$count" -eq "0" ]; then
+        log_info "INFO:  OMS Agent process limit NOT Presently Set."
+    elif [ $count -eq 1 ]; then
+        llimit=`cat $PROC_LIMIT_CONF | grep -E "$LIMIT_LINE_REGEX" | awk '{print $4}'`
+    else
+        log_warning "$count $AGENT_USER entries found in $PROC_LIMIT_CONF.  This may yield problems.  Last entry should apply."
+        log_warning ">>Last entry presumed to apply<<"
+        grep -E "$LIMIT_LINE_REGEX" $PROC_LIMIT_CONF 1>&2
+        log_warning "End of the $count duplicate entry WARNING Message"
+        llimit=`grep -E "$LIMIT_LINE_REGEX" $PROC_LIMIT_CONF | tail -1 | awk '{print $4}'`
+    fi
+
+    if [ "$llimit" -eq -1 ]; then
+        log_info "$AGENT_USER proc count unlimited."
+    elif [ "$llimit" -lt "$MIN_OMSAGENT_PROC_LIMIT" ]; then
+        log_error "$AGENT_USER proc limit setting of '$llimit' is less than minimum of $MIN_OMSAGENT_PROC_LIMIT."
+        resultcode=1
+    fi
+
+    echo $llimit
+    return $resultcode
```

Comment:
This method is a get() call; it should always succeed unless we don't find the file to read from. In my opinion, this should always return successful unless you want to check for a case (which I don't think should occur) of /etc/security/limits.conf not being present.
You log errors when an unexpected case happens, but in those cases you have still successfully shown the current limit.

Reply:
Ok.  I'll add that.  Sounds good.